aliases/available/bash_it.aliases.bash

```diff
@@ -0,0 +1,24 @@
+cite about-alias
+about-alias 'Bash-it aliases'
```

Comment:
I need to change the about of the alias in something more accurate, don't you think?

Reply:
Maybe like this? "Aliases for the bash-it command"

Reply:
Maybe we should also mention here that this file is automatically included from the `general` aliases file.